commit 79e75b3daab517d686d2f6a6737640c84daaf6f3
Author: Peter Jones <pjones@redhat.com>
Date:   Mon Nov 3 10:58:52 2014 -0500

    Make the efi_guid_t objects we export be declared const.
    
    Signed-off-by: Peter Jones <pjones@redhat.com>

diff --git a/src/efivar.h b/src/efivar.h
index e5199e0..5ef8740 100644
--- a/src/efivar.h
+++ b/src/efivar.h
@@ -127,7 +127,7 @@ efi_guid_cmp(const efi_guid_t *a, const efi_guid_t *b)
 	return memcmp(a, b, sizeof (efi_guid_t));
 }
 
-extern efi_guid_t efi_guid_zero;
+extern const efi_guid_t efi_guid_zero;
 
 static inline int
 __attribute__ ((unused))
diff --git a/src/makeguids.c b/src/makeguids.c
index 321a7e7..a822ef2 100644
--- a/src/makeguids.c
+++ b/src/makeguids.c
@@ -27,8 +27,8 @@
 #include "util.h"
 #include "guid.h"
 
-efi_guid_t efi_guid_zero = {0};
-efi_guid_t efi_guid_empty = {0};
+efi_guid_t const efi_guid_zero = {0};
+efi_guid_t const efi_guid_empty = {0};
 
 struct guidname efi_well_known_guids[] = {
 };
@@ -177,10 +177,10 @@ main(int argc, char *argv[])
 		fprintf(symout, "%s_end:\n", outbuf[i].symbol);
 		if (!strcmp(outbuf[i].symbol, "efi_guid_zero")) {
 			fprintf(symout, "efi_guid_empty_end:\n");
-			fprintf(header, "extern efi_guid_t efi_guid_empty;\n");
+			fprintf(header, "extern const efi_guid_t efi_guid_empty;\n");
 		}
 
-		fprintf(header, "extern efi_guid_t %s;\n", outbuf[i].symbol);
+		fprintf(header, "extern const efi_guid_t %s;\n", outbuf[i].symbol);
 	}
 
 	fprintf(header, "\n#endif /* EFIVAR_GUIDS_H */\n");
